app/src/main/java/io/github/viabachelora23michaelkutaibakasper/bprapp/data/repository/EventRepository.kt

```diff
@@ -2,41 +2,42 @@ package io.github.viabachelora23michaelkutaibakasper.bprapp.data.repository
 
 import android.util.Log
 import com.apollographql.apollo3.ApolloClient
-import io.github.viabachelora23michaelkutaibakasper.bprapp.CountryByCodeQuery
-import io.github.viabachelora23michaelkutaibakasper.bprapp.ExampleQuery
+import io.github.viabachelora23michaelkutaibakasper.bprapp.AllPublicEventsQuery
+
 import io.github.viabachelora23michaelkutaibakasper.bprapp.data.domain.Event
+import io.github.viabachelora23michaelkutaibakasper.bprapp.data.domain.GeoLocation
+import io.github.viabachelora23michaelkutaibakasper.bprapp.data.domain.Location
 
 class EventRepository : IEventRepository {
 
-    private val URL = "https://countries.trevorblades.com/"
+    private val URL = "https://api-gateway-6tyymw4cxq-ew.a.run.app/"
```

Comment:
Are there any way to make this more configurable e.g. through some config file or do we just have to edit the repository when the URL changes?

Reply:
Good question. We should look into that

Reply:
You can add them to the manifest, its like appsetting.json in .net

Reply:
Ahh, that could work!